torchmetrics/functional/image/ms_ssim.py

```diff
@@ -0,0 +1,191 @@
+            f"For a given number of `betas` parameters {len(betas)} and kernel size {kernel_size[0]}, the image height "
+            f"must be larger than {(kernel_size[0] - 1) * _betas_div}."
+        )
+    if preds.size()[-1] // _betas_div <= kernel_size[1] - 1:
+        raise ValueError(
+            f"For a given number of `betas` parameters {len(betas)} and kernel size {kernel_size[1]}, the image width "
+            f"must be larger than {(kernel_size[1] - 1) * _betas_div}."
+        )
+
+    for _ in range(len(betas)):
+        sim, contrast_sensitivity = _get_normalized_sim_and_cs(
+            preds, target, kernel_size, sigma, reduction, data_range, k1, k2, normalize
+        )
+        sim_list.append(sim)
+        cs_list.append(contrast_sensitivity)
+        preds = F.avg_pool2d(preds, (2, 2))
+        target = F.avg_pool2d(target, (2, 2))
+
+    sim_stack = torch.stack(sim_list)
+    cs_stack = torch.stack(cs_list)
+
+    if normalize == "simple":
+        sim_stack = (sim_stack + 1) / 2
+        cs_stack = (cs_stack + 1) / 2
+
+    sim_stack = sim_stack ** torch.tensor(betas)
+    cs_stack = cs_stack ** torch.tensor(betas)
+    return torch.prod(cs_stack[:-1]) * sim_stack[-1]
+
+
+def ms_ssim(
+    preds: Tensor,
+    target: Tensor,
+    kernel_size: Sequence[int] = (11, 11),
+    sigma: Sequence[float] = (1.5, 1.5),
+    reduction: str = "elementwise_mean",
+    data_range: Optional[float] = None,
+    k1: float = 0.01,
+    k2: float = 0.03,
+    betas: Tuple[float, float, float, float, float] = (0.0448, 0.2856, 0.3001, 0.2363, 0.1333),
+    normalize: Optional[Literal["relu", "simple"]] = None,
+) -> Tensor:
+    """Computes `MultiScaleSSIM`_, Multi-scale Structual Similarity Index Measure, which is a generalization of
+    Structual Similarity Index Measure by incorporating image details at different resolution scores.
+
+    Args:
+        preds: Predictions from model of shape `[N, C, H, W]`
+        target: Ground truth values of shape `[N, C, H, W]`
+        kernel_size: size of the gaussian kernel (default: (11, 11))
+        sigma: Standard deviation of the gaussian kernel (default: (1.5, 1.5))
+        reduction: a method to reduce metric score over labels.
+
+            - ``'elementwise_mean'``: takes the mean (default)
+            - ``'sum'``: takes the sum
+            - ``'none'``: no reduction will be applied
+
+        data_range: Range of the image. If ``None``, it is determined from the image (max - min)
+        k1: Parameter of SSIM.
+        k2: Parameter of SSIM.
+        betas: Exponent parameters for individual similarities and contrastive sensitivies returned by different image
+        resolutions.
+        normalize: When MS-SSIM loss is used for training, it is desirable to use normalizes to improve the training
+        stability. This `normalize` argument is out of scope of the original implementation [1], and it is adapted from
+        https://github.com/jorge-pessoa/pytorch-msssim instead.
```

Comment:
```suggestion
        normalize: When MS-SSIM loss is used for training, it is desirable to use normalizes to improve the training
            stability. This `normalize` argument is out of scope of the original implementation [1], and it is adapted from
            https://github.com/jorge-pessoa/pytorch-msssim instead.
```
This probably fixes it